### PR TITLE
feat(PLYReader): add basic support for face texture

### PR DESCRIPTION
**Pull request by robbie-li**
For PLY file with face texture, currently vtk.js cannot support the texture coordinates.

I add basic support to convert face texture to points texture.

### CODE REVIEW

**Comment by finetjul** [comment]
Would be great if you can update the example to show the usage of texture on PLY.

**Comment by robbie-li** [comment]
> Would be great if you can update the example to show the usage of texture on PLY.

I add the example, you can select one PLY with one PNG/JPG together.

File: Sources/IO/Geometry/PLYReader/index.js [hunk 1]
```diff
@@ -157,6 +163,27 @@ function postProcess(buffer, elements) {
     }
   }
 
+  if (hasFaceTCoods) {
```
Comment (finetjul) [inline Sources/IO/Geometry/PLYReader/index.js:166]: What if you have both `hasVertTCoords` and `hasFaceTCoods`?

Comment (robbie-li) [reply Sources/IO/Geometry/PLYReader/index.js:166]: I have never seen such type of PLY.
Do you have any suggestion, use VertTCoords or use FaceTCoords?

Comment (daker) [reply Sources/IO/Geometry/PLYReader/index.js:166]: I don't think we can have such case

Comment (robbie-li) [reply Sources/IO/Geometry/PLYReader/index.js]: Anyway, I prefer to take the VertTCoords in this case.

File: Sources/IO/Geometry/PLYReader/index.js [hunk 2]
```diff
@@ -157,6 +163,25 @@ function postProcess(buffer, elements) {
     }
   }
 
+  if (!hasVertTCoords && hasFaceTCoords) {
+    // don't use array.shift, because buffer.indices will be used later
+    let idxVerts = 0;
+    let idxCoord = 0;
+    for (let faceIdx = 0; faceIdx < nbFaces; ++faceIdx) {
+      const nbFaceVerts = buffer.indices[idxVerts++];
+      const texcoods = buffer.faceVertexUvs[idxCoord++];
```
Comment (finetjul) [inline Sources/IO/Geometry/PLYReader/index.js:172]: texcoods -> texcoords

File: Sources/IO/Geometry/PLYReader/index.js [hunk 3]
```diff
@@ -157,6 +163,25 @@ function postProcess(buffer, elements) {
...
+      if (nbFaceVerts * 2 === texcoods.length) {
```
Comment (finetjul) [inline Sources/IO/Geometry/PLYReader/index.js:173]: if it is false, shouldn't you increment idxVerts anyway ?

Comment (robbie-li) [reply Sources/IO/Geometry/PLYReader/index.js]: Yes, you are right.

**Comment by finetjul** [comment]
LGTM

**Comment by daker** [comment]
@jourdain can you please merge if it's ok for you?

**Pull request merged**